source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,684 @@ Filters Builders
+
+   * - ``in()``
+     - any of the values specified in an array.
+
+   * - ``nin()``
+     - none of the values specified in an array.
+
+   * - ``empty()``
+     - all the documents.
+
+
+The following code snippet shows how to use a comparison filter:
+
+.. code-block:: java
+   
+   import static com.mongodb.client.model.Filters.*;
+
+   // <MongoCollection setup code here>
+
+   collection.find(<comparison filter>);
+
+.. example::
+
+   The following example creates a filter that selects all documents where
+   the value of the ``qty`` field equals ``5`` in the paint_purchaces collection:
+
+   .. code-block:: java
+      
+      import static com.mongodb.client.model.Filters.eq;
+
+      // <MongoCollection setup code here>
+
+      Bson equalComparison = eq("qty", 5);
+      List<Document> results = new ArrayList<>();
+      collection.find(equalComparison).into(results);
+      for (Document result : results) {
+         System.out.println(result.toJson());
+      }
+   
+   The Bson renders as:
+
+   .. code-block:: none
+      :copyable: false
+   
+      {
+         "qty": 20
+      }
+
+   The output of the above code snippet should look something like this:
+
+   .. code-block:: json
+      :copyable: false
+   
+      {"_id": 1, "color": "red", "qty": 5, "vendor": ["A"]}
+      {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+
+.. example::
+
+   The following example creates a filter that selects all documents in
+   the paint_purchaces collection because the predicate is empty: 
+
+   .. code-block:: java
+      
+      import static com.mongodb.client.model.Filters.empty
+
+      // <MongoCollection setup code here>
+
+      Bson emptyComparison = empty();
+      List<Document> results = new ArrayList<>();
+      collection.find(emptyComparison).into(results);
+      for (Document result : results) {
+         System.out.println(result.toJson());
+      }
+   The Bson renders as:
+
+   .. code-block:: none
+      :copyable: false
+   
+      { }
+
+   The output of the above code snippet should look something like this:
+   
+   .. code-block:: json
+      :copyable: false
+      
+      {"_id": 1, "color": "red", "qty": 5, "vendor": ["A"]}
+      {"_id": 2, "color": "purple", "qty": 10, "vendor": ["C", "D"]}
+      {"_id": 3, "color": "blue", "qty": 8, "vendor": ["B", "A"]}
+      {"_id": 4, "color": "white", "qty": 6, "vendor": ["D"]}
+      {"_id": 5, "color": "yellow", "qty": 11, "vendor": ["A", "B"]}
+      {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+      {"_id": 7, "color": "green", "qty": 8,"vendor": ["B", "C"]}
+      {"_id": 8, "color": "orange", "qty": 7, "vendor": ["A", "D"]}
+
+.. _logical:
+
+Logical
+-------
+
+The logical operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Logical Operator
+     - 
+      
+   * - ``and()``
+     - Joins filters with a logical AND and selects documents that match the conditions of both filters.
```

Comment:
Nitpick: we could use monospace for the logical operators to identify them as terms rather than grammatical conjunctions.